### PR TITLE
Bug 1129630 - Add RTL support to the date picker r=alive

### PR DESCRIPTION
**Pull request by crh0716**
https://bugzilla.mozilla.org/show_bug.cgi?id=1129630

### CODE REVIEW

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0ceb9cfdd937)

File: shared/style/date_selector.css [hunk 1]
```diff
@@ -277,3 +280,53 @@ button[disabled]::-moz-selection {
 [role="dialog"][data-type="date-selector"] menu button.full {
   width: 100%;
 }
+
+/* Right-To-Left layout */
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .picker-bar-background {
+  left: 0;
+  right: auto;
```
Comment (fabi1cazenave) [inline shared/style/date_selector.css]: in this case, please use `unset` instead of `auto` (same for the following lines)

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=82ee014198e9)

**Comment by try-server-hook** [comment]
[crh0716 Arthur Chen (crh0716)](https://github.com/crh0716) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f81a1a9d784f)

File: shared/style/date_selector.css [hunk 2]
```diff
@@ -277,3 +277,53 @@ button[disabled]::-moz-selection {
...
+}
+
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .value-picker-month-wrapper {
+  left: unset;
+  right: 21%;
+}
+
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .value-picker-year-wrapper {
+  left: 0;
+  right: auto;
```
Comment (fabi1cazenave) [inline shared/style/date_selector.css]: nit: you probably meant `unset` ;-)

File: shared/style/date_selector.css [hunk 3]
```diff
@@ -277,3 +277,53 @@ button[disabled]::-moz-selection {
...
+}
+
+/* YMD */
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .YMD .value-picker-year-wrapper {
+  left: auto;
```
Comment (fabi1cazenave) [inline shared/style/date_selector.css]: same here

File: shared/style/date_selector.css [hunk 4]
```diff
@@ -277,3 +277,53 @@ button[disabled]::-moz-selection {
...
+}
+
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .YMD .value-picker-month-wrapper  {
+  left: unset;
+  right: 27%;
+}
+
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .YMD .value-picker-date-wrapper {
+  left: 0;
+  right: auto;
```
Comment (fabi1cazenave) [inline shared/style/date_selector.css]: and here

**Pull request merged**